core/executor.py

```
@@ -651,40 +879,172 @@ def save_execution_result(self, decisions: list, artifacts: list) -> None:
+        user_answer_lower = user_answer.lower()
+
+        if any(keyword in user_answer_lower for keyword in confirmation_keywords):
+            # 如果是确认，返回一个通用的确认决策
+            return "用户确认需求"
+
+        # 检查是否包含决策关键词
+        decision_keywords = ["决定", "选择", "采用", "使用", "需要", "要"]
+        for keyword in decision_keywords:
+            if keyword in user_answer:
+                # 提取包含关键词的句子
+                sentences = user_answer.split("，|。|；|！")
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)